app/controllers/rewards_controller.rb

```diff
@@ -1,8 +1,9 @@
 class RewardsController < ApplicationController
   before_action :authenticate_employee!
   def index
-    pagy, rewards = pagy(Reward.order(created_at: :desc), items: 3)
-    render :index, locals: { rewards: rewards, pagy: pagy }
+    pagy, rewards = pagy(FilterRewards.new(Reward.order(created_at: :desc)
```

Comment:
I would use here a local variable so the code is easier to read.

```suggestion
filtered_rewards = FilterRewards.new(Reward.order(created_at: :desc)(...)
    pagy, rewards = pagy(filtered_rewards)
```